decisions.py

```diff
@@ -51,13 +51,14 @@ def __init__(self, publisher_msg, publishing_topic, qos_publisher, rate=10, moti
 
         
         if motion_type==POINT_PLANNER:
-            self.controller=controller(klp=0.2, klv=0.5, kap=0.8, kav=0.6)      
+            # Lab 2: klp=0.1, klv=0.5, kli=0.5, kap=1.5, kav=0.6, kai=0.5
+            self.controller=controller(klp=0.2, klv=0.5, kap=0.8, kav=0.6)
             self.planner=planner(POINT_PLANNER)
 
         
         elif motion_type==TRAJECTORY_PLANNER:
-            # TODO PART 5 Bonus Put the gains that you conclude from lab 2
-            self.controller=trajectoryController(...)      
+            # PART 5 Bonus Put the gains that you conclude from lab 2
+            self.controller=trajectoryController(klp=0.1, klv=0.5, kli=0.5, kap=1.5, kav=0.6, kai=0.5)
```

Comment:
Took this from 
<img width="779" alt="image" src="https://github.com/user-attachments/assets/24effa3d-b040-42bb-8f3f-10d9c48d79a0">

which is from 
<img width="386" alt="image" src="https://github.com/user-attachments/assets/dd7b6be8-a1e3-4942-8378-3323015776d2">